Unify space removal info on A&DD page [DOC-1014] (#7257)
segmentio/segment-docs: src/privacy/account-deletion.md

## Remove a Unify space

You can remove a Unify space from your workspace by disabling it, which hides the space from the Segment UI, or by deleting it, which removes all data about that space from Segment's servers. Disabled or deleted spaces do not count toward your [Unify limits](/docs/unify/product-limits/). 

> warning "You cannot reuse space names"
> Do not use the names of previously disabled or deleted spaces for new Unify spaces.

### Disable a Unify space

Disabled spaces are not accessible from the Segment app UI, but all data from your space remains on Segment's servers. Disabled spaces do not count toward your Unify space limit, and can be re-enabled by sending an email to the Customer Success team [(friends@segment.com)](mailto:friends@segment.com).

To disable a Unify space, email the Customer Success team [(friends@segment.com)](mailto:friends@segment.com) to create a support ticket. In your email to Customer Success, include the following information:
 - Workspace slug
 - Unify space ID

> ### Disable an Engage space

Author: ```suggestion
### Delete an Engage space
```

> note "Deleting source data"
> When Segment deletes your data for a particular source, the deletion is not forwarded to sources or data storage providers associated with your account: your data is only removed from Segment's S3 archive buckets. To remove your data from external sources, reach out to the individual source about their deletion practices. 

## Remove a Unify space

> You can remove a Unify space from your workspace by disabling it, which hides the space from the Segment UI, or by deleting it, which removes all data about that space from Segment's servers. Disabled or deleted spaces do not count toward your [Unify limits](/docs/unify/product-limits/). 

Reviewer: ```suggestion
You can remove a Unify space from your workspace by disabling it, which hides the space from the Segment UI, or by deleting it, which removes all data about that space from Segment's servers. Disabled or deleted spaces don't count toward your [Unify limits](/docs/unify/product-limits/). 
```

> > warning "You cannot reuse space names"

Reviewer: ```suggestion
> warning "You can't reuse space names"
```

> > Do not use the names of previously disabled or deleted spaces for new Unify spaces.

Reviewer: ```suggestion
> Don't use the names of previously disabled or deleted spaces for new Unify spaces.
```

### Disable a Unify space

> Disabled spaces are not accessible from the Segment app UI, but all data from your space remains on Segment's servers. Disabled spaces do not count toward your Unify space limit, and can be re-enabled by sending an email to the Customer Success team [(friends@segment.com)](mailto:friends@segment.com).

Reviewer: ```suggestion
Disabled spaces aren't accessible from the Segment app UI, but all data from your space remains on Segment's servers. Disabled spaces don't count toward your Unify space limit, and can be re-enabled by sending an email to the Customer Success team [(friends@segment.com)](mailto:friends@segment.com).
```

To disable a Unify space, email the Customer Success team [(friends@segment.com)](mailto:friends@segment.com) to create a support ticket. In your email to Customer Success, include the following information:
 - Workspace slug
 - Unify space ID

> success ""
> You can find your Unify space ID by navigating to the Unify settings page and selecting **API Access**.

### Delete a Unify space

> Deleted spaces, and all data associated with a deleted space, are permanently removed from Segment's servers. You cannot recover a deleted space. 

Reviewer: ```suggestion
Deleted spaces, and all data associated with a deleted space, are permanently removed from Segment's servers. You can't recover a deleted space. 
```
